### PR TITLE
union of two unsorted arrays  Issue:#3715

### PR DESCRIPTION
**Pull request by jaimehta19**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Comment by jaimehta19** [comment]
@ananddasani please review

**Comment by ananddasani** [comment]
@jaimehta1912 resolve this conflict

File: C-Plus-Plus/cp/Tug_of_water.cpp [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#include<iostream>
```
Comment (ananddasani) [inline C-Plus-Plus/cp/Tug_of_water.cpp:9]: Question is :: union of two unsorted arrays
and you have submitted the code of :: Tug_of_war ??

please check again everything :)

**Comment by ananddasani** [comment]
check the source code file once

**Review by rohanbabbar04 — COMMENTED** [review]
Create different branches to solve different issues

**Comment by rohanbabbar04** [comment]
@jaimehta1912 do no merge from the main branch create a new branch to work on a new issue.
Create multiple branches to work on different issues

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Don't push more than one code file in a single PR

**Pull request closed without merging**